apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/DeployNewReplicaPanel.tsx

```diff
@@ -78,9 +78,11 @@ const DeployNewReplicaPanel = ({
   const { data: subscription } = useOrgSubscriptionQuery({ orgSlug: org?.slug })
   const { data: diskConfiguration } = useDiskAttributesQuery({ projectRef })
 
-  const hasOverdueInvoices =
-    (allOverdueInvoices ?? []).filter((x) => x.organization_id === project?.organization_id)
-      .length > 0
+  const overdueInvoices = (allOverdueInvoices ?? []).filter(
+    (x) => x.organization_id === project?.organization_id
+  )
+  const ignoreOverdueInvoices = ['team', 'enterprise'].includes(subscription?.plan.id ?? '')
```

Comment:
I think it would be neater if we simply disabled the entire overdueInvoices call for these customers (enabled flag on the query), so there is no unnecessary network call

Reply:
gotcha just pushed a change! sorry only just saw this comment 🙏